tasks/depcheck.js

```diff
@@ -16,24 +16,32 @@ module.exports = function (grunt) {
   grunt.registerMultiTask('depcheck', 'Depcheck Grunt plugin', function () {
     var options = this.options({
       'withoutDev': false,
+      'failOnUnusedDeps': false,
       'ignoreDirs': ['.git','.svn','.hg','.idea','node_modules','bower_components'],
       'ingoreMatches': []
     });
 
     var dirsChecked = 0;
     var done = this.async();
+    var fail = false;
     var files = this.filesSrc;
 
     files.forEach(function (f) {
-      depcheck.check(path.resolve(f), options, function (unused) {
+      depcheck.check(path.resolve(f), {
+        ignoreDirs: options.ignoreDirs,
+        ingoreMatches: options.ingoreMatches,
+        withoutDev: options.withoutDev
```

Comment:
I think it is fine to directly pass the `options` to depcheck. I don't think we are going to use `options.failOnUnusedDeps` in depcheck.